shared/typings/torngit.py

```diff
@@ -25,9 +30,13 @@ class GithubInstallationInfo(TypedDict):
     app_id: Optional[int]
     pem_path: Optional[str]
 
+class AdditionalData(TypedDict):
+    upload_type: Optional[UploadType]
```

Comment:
When you say "Optional" this indicates that the key will always be there.
Looking above the initialization is just `{}` (no key).

If the key may or may not be present then use `NotRequired` (https://docs.python.org/3/whatsnew/3.11.html#pep-655-marking-individual-typeddict-items-as-required-or-not-required)

I'm sure other fields in this file suffer from the same problem 😅

Reply:
We have an `UploadType` that's the DB Enum. We do have a ReportType in Worker that has what I'm looking for, so I'll add that to this change

And will add the `NotRequired`, TIL